### DISCUSSION TITLE
Multiple models at the same time.

### DISCUSSION THREAD

**Discussion by Biglow230** [google/model-viewer#3028] [Q&A]
Is there a way to display multiple .glb/gltf at the same time.

Similar to https://threejs.org/examples/#webgl_animation_multiple

I have 4 sets of models that come together but are interchangeable, I'd like to make a product configurator in which parts can be swapped out.

 Example
Part a.glb 
Part a1.glb
Part a2.glb
Part a3.glb

**Comment by elalish** [comment]
No, this starts to  open a fairly large can of worms. We're trying to keep MV simple and understandable. For this level of detail you'll  want a game engine like three.js. You're welcome to fork us and add what you need  in the  three.js layer.
